python/PyQt6/gui/auto_generated/codeeditors/qgscodeeditorwidget.sip.in

```diff
@@ -102,6 +102,17 @@ Returns the widget's associated file path.
 .. seealso:: :py:func:`setFilePath`
 
 .. seealso:: :py:func:`filePathChanged`
+%End
+
+    bool save( const QString &path = QString() );
+%Docstring
+Saves the code editor content into the file ``path``.
+
```

Comment:
This needs some explanation like "returns false if the file path has not previously been set, or if writing the file fails."

Reply:
@nyalldawson , done.